manifests/shared/deployments/datadog.yml

```diff
@@ -0,0 +1,14 @@
+---
+releases:
```

Comment:
But good point is that now we are uploading the release in 2 ways, since the release with URL gets included in CF manifest (even though it is not needed there, but we merge the datadog file so that we can retrieve meta), so it will get uploaded. I'll remove the git resource and build&upload part then. This way concourse, bosh and all CF VMs will always deploy the same version.